### ISSUE TITLE
deposit: geographical metadata

### ISSUE THREAD

**Issue by slint** [zenodo/zenodo#1744]
**Data model**

```python
{
    "locations": [
        {
            "lat": "31.32", 
            "lon": "33.4343", 
            "place": "Some place", 
            "description": "A textual description"
        },
    ],
}
```

- `place` and `description` are free-form text
- Only store a list of points (i.e no boxes or polygons)
- Validate latitude and longitude

**Comment by TimSmithCH** [comment]
Naive question: shouldn't this be 3D ie include altitude?

**Comment by slint** [comment]
Closed via #1751
